### PR TITLE
Logging to DB after DB is initialized

### PR DESCRIPTION
**Pull request by seanmcilvenna**

### CODE REVIEW

File: core/src/main/java/com/lantanagroup/link/db/SharedService.java [hunk 1]
```diff
@@ -59,6 +62,26 @@ public Connection getSQLConnection() {
...
+
+    DBAppender appender = new DBAppender();
+    appender.setContext(logCtx);
+    appender.setConnectionSource(source);
+    appender.setName("link-db");
+    appender.start();
+
+    logCtx.getLogger("ROOT").addAppender(appender);
+
+    logger.warn("test");
```
Comment (smailliwcs) [inline core/src/main/java/com/lantanagroup/link/db/SharedService.java:82]: Assume this test message can be removed?

File: pom.xml [hunk 2]
```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>com.microsoft.sqlserver</groupId>
             <artifactId>mssql-jdbc</artifactId>
-            <version>12.4.2.jre11</version>
+            <version>12</version>
```
Comment (smailliwcs) [inline pom.xml:68]: This causes a dependency resolution error for me.  Are you able to `mvn clean compile` with this in place?

**Review by smailliwcs — CHANGES_REQUESTED** [review]
FWIW, in my testing, Logback seemed robust enough to keep plugging away despite a nonexistent database, then start using it once `shared-db.sql` was applied.  But definitely preferable this way, if for no other reason than to reduce the log noise associated with the errors that would get thrown.

File: pom.xml (hunk 2 above)
Comment (smailliwcs) [reply pom.xml:68]: Looks like the CI pipeline also failed because of this.

Comment (seanmcilvenna) [reply pom.xml:68]: This was definitely a mistake. Didn't mean for this. I'm rolling back the change now.

File: core/src/main/java/com/lantanagroup/link/db/SharedService.java (hunk 1 above)
Comment (seanmcilvenna) [reply core/src/main/java/com/lantanagroup/link/db/SharedService.java]: Yup, removed.

**Review by smailliwcs — APPROVED** [review]

**Pull request merged**